src/AltinnRepositories/gitea-data/gitea/templates/base/head.tmpl

```diff
@@ -154,7 +154,7 @@
 						<a class="item{{if .PageIsIssues}} active{{end}}" href="{{AppSubUrl}}/issues">{{.i18n.Tr "issues"}}</a>
 						<a class="item{{if .PageIsPulls}} active{{end}}" href="{{AppSubUrl}}/pulls">{{.i18n.Tr "pull_requests"}}</a>
 					{{else}}
-						<a class="item{{if .PageIsHome}} active{{end}}" href="{{AppSubUrl}}/">{{.i18n.Tr "home"}}</a>
+						<a class="item{{if .PageIsHome}} active{{end}}" href="{{AppSubUrl}}/designer">{{.i18n.Tr "home"}}</a>
```

Comment:
not sure this change is necessary.. Testing locally if it can be removed or not.

Reply:
"/designer" is routed to "designer/designer" in nginx, while "/" is just "designer" so I dont think this should be changed.

Reply:
but nothing should actually be changed for designer with our changes, so why is this required now? We haven't really udpdated anything to do with the designer routing, or??

Reply:
Yes, Appsuburl which is {environment}/repos - which again loads the home/index. So i think, that can be removed but i agree that testing has to be done locally

Reply:
I agree with you @acn-sbuad , this should not be changed. To be clear: keep it as "/".

Reply:
aah, got it! Build in progress to confirm that it should be removed.